Maple2.Server.Game/Packets/GuildPacket.cs

```diff
@@ -706,7 +706,7 @@ private enum Command : byte {
             pWriter.WriteLong(application.Guild.Id);
             pWriter.WriteUnicodeString(application.Guild.Name);
             pWriter.WriteUnicodeString(application.Guild.Emblem);
-            pWriter.Write<Trophy>(application.Guild.Trophy);
+            pWriter.WriteClass<Maple2.Model.Game.AchievementInfo>(application.Guild.AchievementInfo);
```

Comment:
why do we need the whole `Maple2.Model.Game.AchievementInfo` here? can we not import it?